biot_python_sdk/biot.py

```
@@ -434,7 +434,7 @@ def upload_file(self, file_path):
         else:
             return None
     
-    def upload_file_from_ram(self, data, file_name):
+    def upload_file_from_ram(self, data, file_name,mime_type = 'application/octet-stream'):
```

Ruff (0.8.2)

Missing return type annotation for public function `upload_file_from_ram`

(ANN201)